src/main/java/ua/ithillel/bank/versioning/AccountController.java

```diff
@@ -0,0 +1,46 @@
+package ua.ithillel.bank.versioning;
+
+import org.springframework.web.bind.annotation.*;
+import ua.ithillel.bank.versioning.service.AccountDto;
+import ua.ithillel.bank.versioning.service.AccountService;
+
+import java.util.List;
+
+@RestController
+public class AccountController {
+    private final AccountService accountService;
+
+    public AccountController(AccountService accountService) {
+        this.accountService = accountService;
+    }
+
+    @GetMapping("/api/accounts")
+    public List<AccountDto> getAllAccounts() {
+        return accountService.getAllAccounts();
+    }
+
+    @GetMapping("/api/persons/{uid}/accounts")
+    public List<AccountDto> getUidAccounts(@PathVariable("uid") String uid) {
+        return accountService.getAccounts(uid);
+    }
+
+    @GetMapping("/api/accounts/{uid}")
+    public AccountDto getAccountByIBan(@PathVariable("uid") String uid) {
+        return accountService.getAccount(uid);
+    }
+
+    @PostMapping("/api/persons/{uid}/accounts")
+    public AccountDto createAccount(@PathVariable("uid") String uid, @RequestBody AccountDto account) {
```

Comment:
```suggestion
    public AccountDto createAccount(@PathVariable("uid") String personUid, @RequestBody AccountDto account) {
```
коли два схожих id (personId та accountId) ходять разом, то краще розрізняти імена